design/Geth_type_conventions.md

```diff
@@ -0,0 +1,48 @@
+# Code Conventions Proposal - Geth Type Renames
+
+This proposal seeks to establish a standard on how the enclave codebase should use the standard `geth` types like `types.Transaction`, `types.Block` and the likes.
```

Comment:
why limiting it to the enclave, and not the entire codebase?

Reply:
I haven't went through everything in the code base, thus I'm not sure if the problem exists there. We can make it for the whole codebase.